.github/workflows/ci.yml

```diff
@@ -83,3 +83,7 @@ jobs:
     - name: Test
       run: |
         cabal test all
+    # ----------------
+    - name: Bench
+      run: |
+        cabal bench all --benchmark-options='--list-tests'
```

Comment:
Running benchmarks on the cloud I think is pure waste of CPU time. Too many jobs running on same hardware, too many layers of virtualization. I think it's enough to check that benchmarks build successfully. `Build` step already do that

Reply:
I think it is useful at least to list benchmarks. 
Updated with tweaked `cabal configure` options, to avoid redundant reconfiguration.